renderer.js

```diff
@@ -2,6 +2,30 @@
 // JQuery
 var $ = require("jquery");
 
+// Paging
+function paging(group, i){
+  var img = $(group+' img');
+  console.log(img);
+  img.each(function (index, el) {
+    if (index>=i*70 && index<((+i+1)*70)) {
```

Comment:
You should pay attention to your coding style. I think  `index >= i * 70 ` more clearer than `index>=i*70`.  
FYI: [Google JavaScript Style Guide](https://google.github.io/styleguide/javascriptguide.xml) I am not sure this is best. Maybe you can find better one. try it!